src/sender.js

```diff
@@ -386,20 +532,39 @@ class Sender {
         write(this, ' ');
         write(this, timestampStr);
         write(this, '\n');
+        this.pendingRowCount++;
         startNewRow(this);
+        await autoFlush(this);
     }
 
     /**
      * Closing the row without writing designated timestamp into the buffer of the sender. <br>
      * Designated timestamp will be populated by the server on this record.
      */
-    atNow() {
+    async atNow() {
```

Comment:
Sync APIs in node should exist/be used only for small short-lived scripts. Otherwise, calling them will block the event loop leading to starvation of pretty much everything that was running in the node program. So long story short, we should only support async APIs (async/await, promise, callback-based, doesn't matter) on Node.js.